client/client.go

```diff
@@ -55,6 +55,16 @@ func pemEncodeCert(contents string) string {
 	return strings.Join([]string{"-----BEGIN CERTIFICATE-----", contents, "-----END CERTIFICATE-----"}, "\n")
 }
 
+// convert address to localhost when no ip is specefied
```

Comment:
Comments that are complete sentences should be capitalized and punctuated like standard English sentences.

typo: specified